docs/content/how-tos/rules/FL0076.md

```diff
@@ -0,0 +1,27 @@
+---
+title: FL0076
+category: how-to
+hide_menu: true
+---
+
+# FavourStaticEmptyFields (FL0076)
+
+*Introduced in `0.21.1`*
+
+## Cause
+
+Rule favours static empty fields such as String.Empty, List.Empty and also Array.Empty.
+
+## Rationale
+
+Using static empty fields aids readibility.
+
+## How To Fix
+
+Use a static empty field such as String.Empty or List.Empty and Array.Empty.
```

Comment:
`a or b and c`? I guess you mean `a, b or c` or `a, b and c`, but not a mix.

Reply:
Done. Thanks.